qcodes/instrument_drivers/american_magnetics/AMI430.py

```diff
@@ -462,9 +463,9 @@ def _update_units(
 class AMI430_3D(Instrument):
     def __init__(self,
                  name: str,
-                 instrument_x: AMI430,
-                 instrument_y: AMI430,
-                 instrument_z: AMI430,
+                 instrument_x: Union[AMI430, str],
+                 instrument_y: Union[AMI430, str],
+                 instrument_z: Union[AMI430, str],
                  field_limit: Union[numbers.Real,
                                     Iterable[CartesianFieldLimitFunction]],
                  **kwargs: Any):
```

Comment:
Should this have a docstring describing the use of other instruments as str etc ...?

Reply:
abolutely, i forgot, will do in a subsequent PR